src/styles/base/_column.scss

```diff
@@ -0,0 +1,29 @@
+.column-half-media{
+	@media (min-width: 1224px) {
```

Comment:
I think the indentation is being shown incorrectly on Github's page. This file shows how alignment is correct: https://github.com/joaquinTop/targetMVD-web-react/blob/ui_improvements/src/styles/base/_column.scss